quickjs.c

```diff
@@ -1273,6 +1273,8 @@ static JSValue js_promise_resolve(JSContext *ctx, JSValue this_val,
                                   int argc, JSValue *argv, int magic);
 static JSValue js_promise_then(JSContext *ctx, JSValueConst this_val,
                                int argc, JSValueConst *argv);
+static BOOL js_string_eq(JSContext *ctx,
+                             const JSString *p1, const JSString *p2);
```

Comment:
```suggestion
static BOOL js_string_eq(JSContext *ctx,
                         const JSString *p1, const JSString *p2);
```
But you might as well drop the unused JSContext argument:
```suggestion
static BOOL js_string_eq(const JSString *p1, const JSString *p2);
```

Reply:
Right! Do you want me to do so for `js_string_compare` as well? (I originally wanted to stay close to that signature)

Reply:
You can if you want to, either way is fine.